### ISSUE TITLE
Suggestion: c.RenderError(err) to also log the error

### ISSUE THREAD

**Issue by chrishulbert** [revel/revel#1466]
Hi all, first up: Revel is great!

I just spend a good chunk of time trying to figure out what was wrong in one of my deployed revel apps. I discovered that if your controller does `return c.RenderError(err)` it *doesn't* get logged.

The default 500.html contains 'This exception has been logged' - so i found this surprising that errors *aren't* logged by default.

Would you please consider adding a call to eg `c.Log.Error(err.Error())` to the top of RenderError? I believe it'd follow the 'principle of least surprise' to do so :)

Have a great week all.

**Comment by ptman** [comment]
I think the status 500 probably is logged? But sure logging the error would make sense as well

**Comment by chrishulbert** [comment]
You're right, the 500 is logged, but not the exception/error details (unless i'm mistaken) so i think this could be helpful.
